cfgov/cfgov/urls.py

```diff
@@ -542,6 +551,39 @@ def empty_200_response(request, *args, **kwargs):
 
 ]
 
+# Ask CFPB search and autocomplete
+ask_urlpatterns = [
+    re_path(
+        r'^ask-cfpb/search/$',
+        ask_search,
+        name='ask-search-en'
+    ),
+    re_path(
+        r'^ask-cfpb/search/(?P<as_json>json)/$',
+        ask_search,
+        name='ask-search-en-json'
+    ),
+    re_path(
+        r'^(?P<language>es)/obtener-respuestas/buscar/$',
+        ask_search,
+        name='ask-search-es'
+    ),
+    re_path(
+        r'^(?P<language>es)/obtener-respuestas/buscar/(?P<as_json>json)/$',
+        ask_search,
+        name='ask-search-es-json'
+    ),
+    re_path(
+        r'^ask-cfpb/api/autocomplete/$',
+        ask_autocomplete, name='ask-autocomplete-en'
+    ),
+    re_path(
+        r'^(?P<language>es)/obtener-respuestas/api/autocomplete/$',
+        ask_autocomplete, name='ask-autocomplete-es'
+    ),
+]
+urlpatterns = urlpatterns + ask_urlpatterns
+
```

Comment:
I don't think I understand the point of adding these URLs here. It overrides the flagged URLs you've got above.

Reply:
Removed after adding fallback to flagged_re_path

Reply:
Removed as I added fallback to flagged_re_path